lms/urls.py

```diff
@@ -101,6 +101,7 @@
 
     url(r'^api/commerce/', include('commerce.api.urls', namespace='commerce_api')),
     url(r'^api/credit/', include('openedx.core.djangoapps.credit.urls', app_name="credit", namespace='credit')),
+    url(r'^rss_proxy/', include('rss_proxy.urls', namespace='rss_proxy')),
```

Comment:
I guess my question is what is the value add to an Open edX operator of a default off feature flag that wraps the rss_proxy endpoint? I understand the concept of hiding things behind feature flags, but shouldn't there be some added value in doing so? Otherwise, I feel like we are just adding code which doesn't really do much for anyone and adds complexity (granted, a small amount in this case) to configuration management.